### ISSUE TITLE
Changing Cropping Doesn’t Regenerate Thumbnails

### ISSUE THREAD

**Issue by alapiere** [Rahe/Simple-image-sizes#49]
Hello!

First of all: What a fantastic plugin! I’ve used it on so many websites. Congratulations on the great work. 🙂

I found there’s a bug : If I change the cropping parameters for a thumbnail, as an example: from center-center to center-top and try to regenerate the thumbnails I get the following messages:

This file already exists in this size and have not been regenerated

The only way around this that I found is turning the cropping to “No”, regenerating the thumbnails, and then setting the cropping to “center-top” and regenerating again.

Can we find a way to make this a little more direct without taking the extra step?

**Comment by MikeWhelan** [comment]
I agree, thumbnails should be regenerated when crop settings are changed.

**Comment by Klemart3D** [comment]
@alapiere your workaround not works for me: by choosing "No" and regenerate thumbnails works but by choosing “center-top” and regenerating, images going back to previous cropping, like if it was "yes" or "center-center" :/
